### ISSUE TITLE
Exception showing status.

### ISSUE THREAD

**Issue by pchote** [warwick-one-metre/environmentd#9]
```
saft@sqftnuc2:~> environment status
Traceback (most recent call last):
  File "/usr/bin/environment", line 202, in <module>
    sys.exit(print_status())
  File "/usr/bin/environment", line 48, in print_status
    status = environment.status()
  File "/usr/lib/python3.4/site-packages/Pyro4/core.py", line 171, in __call__
    return self.__send(self.__name, args, kwargs)
  File "/usr/lib/python3.4/site-packages/Pyro4/core.py", line 426, in _pyroInvoke
    raise data
AttributeError: 'NoneType' object has no attribute 'strftime'
 +--- This exception occured remotely (Pyro) - Remote traceback:
 | ----------------------------------------------------
 |  EXCEPTION <class 'AttributeError'>: 'NoneType' object has no attribute 'strftime'
 |  Extended stacktrace follows (most recent call last)
 | ----------------------------------------------------
 | File "/usr/lib/python3.4/site-packages/Pyro4/core.py", line 1162, in Daemon.handleRequest
 | Source code:
 |     data = method(*vargs, **kwargs)  # this is the actual method call to the Pyro object
 | ----------------------------------------------------
 | File "/usr/bin/environmentd", line 315, in EnvironmentDaemon.status
 | Source code:
 |     obs, data = watcher.status()
 | Local values:
 |     can_observe = True
 |     data = {'superwasp_queue_start': '2016-02-13T19:19:02Z', 'superwasp_measurement_start': '2016-02-13T19:21:48Z', 'superwasp_measurement_end': '2016-02-13T19:41:42Z', 'superwasp_sky_temp': (-48.3, -41.2, True), 'superwasp_sufficient_data': True, 'superwasp_queue_len': 132, 'superwasp_ext_humidity': (19.0, 21.0, True), 'superwasp_dew_point_delta': (21.32, 22.51, True)}
 |     obs = True
 |     ret = {'roomalert_internal_humidity': (18.04, 21.24, True), 'roomalert_trap_closed': [True], 'roomalert_internal_temp': (10.0, 11.3, True), 'vaisala_measurement_end': '2016-02-13T19:41:39Z', 'roomalert_roomalert_humidity': (11.5, 14.09, True), 'roomalert_hatch_closed': [True], 'vaisala_wind_speed': (6.1, 21.3, True), 'superwasp_sky_temp': (-48.3, -41.2, True), 'vaisala_relative_humidity': (21.3, 23.6, True), 'roomalert_measurement_end': '2016-02-13T19:41:43Z', 'superwasp_queue_len': 132, 'vaisala_queue_len': 132, 'roomalert_sufficient_data': True, 'superwasp_dew_point_delta': (21.32, 22.51, True), 'vaisala_temperature': (11.2, 11.6, True), 'roomalert_truss_temp': (11.87, 14.69, True), 'vaisala_sufficient_data': True, 'roomalert_measurement_start': '2016-02-13T19:21:47Z', 'vaisala_queue_start': '2016-02-13T19:19:44Z', 'superwasp_queue_start': '2016-02-13T19:19:02Z', 'roomalert_queue_len': 132, 'superwasp_measurement_end': '2016-02-13T19:41:42Z', 'superwasp_measurement_start': '2016-02-13T19:21:48Z', 'roomalert_roomalert_temp': (13.2, 16.0, True), 'roomalert_queue_start': '2016-02-13T19:18:10Z', 'superwasp_sufficient_data': True, 'vaisala_measurement_start': '2016-02-13T19:21:54Z', 'superwasp_ext_humidity': (19.0, 21.0, True), 'vaisala_pressure': (779.9, 780.1, True)}
 |     self = <__main__.EnvironmentDaemon object at 0x7f8712c2ab38>
 |         self._watchers = [<__main__.PyroWatcher object at 0x7f8712c2ae48>, <__main__.PyroWatcher object at 0x7f8712c37cc0>, <__main__.PyroWatcher object at 0x7f8712c37f60>, <__main__.PyroWatcher object at 0x7f8712c37a90>]
 |         self._pyroId = 'environment_daemon'
 |         self._pyroDaemon = <Pyro4.core.Daemon at 0x7f8712bae5c0, 192.168.0.102:9002, 2 objects>
 |     watcher = <__main__.PyroWatcher object at 0x7f8712c37a90>
 | ----------------------------------------------------
 | File "/usr/bin/environmentd", line 280, in PyroWatcher.status
 | Source code:
 |     measurement_start.strftime('%Y-%m-%dT%H:%M:%SZ'),
 | Local values:
 |     data = {'diskspace_queue_len': 4, 'diskspace_sufficient_data': False, 'diskspace_data_fs_available_bytes': (0, False), 'diskspace_data_fs_percent_available': (0, False)}
 |     date = datetime.datetime(2016, 2, 13, 19, 18, 49)
 |     max_first_date = datetime.datetime(2016, 2, 13, 19, 21, 45, 815870)
 |     measurement = {'data_fs_total_bytes': 117664907264, 'data_fs_available_bytes': 31602171904, 'date': datetime.datetime(2016, 2, 13, 19, 18, 49), 'data_fs_percent_available': 27}
 |     measurement_end = datetime.datetime(2016, 2, 13, 19, 18, 49)
 |     measurement_start = None
 |     measurements_in_window = []
 |     min_last_date = datetime.datetime(2016, 2, 13, 19, 36, 45, 815871)
 |     param = <__main__.AggregateParameter object at 0x7f8712bae588>
 |     param_key = 'diskspace_data_fs_percent_available'
 |     param_safe = False
 |     param_value = (0, False)
 |     queue_len = 4
 |     queue_start = datetime.datetime(2016, 2, 13, 19, 12, 48)
 |     safe = False
 |     self = <__main__.PyroWatcher object at 0x7f8712c37a90>
 |         self._parameters = [<__main__.AggregateParameter object at 0x7f8712bae550>, <__main__.AggregateParameter object at 0x7f8712bae588>]
 |         self._query_delay = 120
 |         self._pyro_uri = 'PYRO:diskspace_daemon@192.168.0.101:9008'
 |         self._max_data_gap = datetime.timedelta(0, 300)
 |         self._data = deque([{'data_fs_total_bytes': 117664907264, 'data_fs_available_bytes': 31741972480, 'date': datetime.datetime(2016, 2, 13, 19, 12, 48), 'data_fs_percent_available': 27}, {'data_fs_total_bytes': 117664907264, 'data_fs_available_bytes': 31741972480, 'date': datetime.datetime(2016, 2, 13, 19, 14, 48), 'data_fs_percent_available': 27}, {'data_fs_total_bytes': 117664907264, 'data_fs_available_bytes': 31741972480, 'date': datetime.datetime(2016, 2, 13, 19, 16, 48), 'data_fs_percent_available': 27}, {'data_fs_total_bytes': 117664907264, 'data_fs_available_bytes': 31602171904, 'date': datetime.datetime(2016, 2, 13, 19, 18, 49), 'data_fs_percent_available': 27}], maxlen=11)
 |         self._daemon_name = 'diskspace'
 |     sufficient_data = False
 | ----------------------------------------------------
 |  EXCEPTION <class 'AttributeError'>: 'NoneType' object has no attribute 'strftime'
 | ----------------------------------------------------
 +--- End of remote traceback
```

**Comment by pchote** [comment]
This happens when `measurement_start` is null, which happened here because the disk space daemon wasn't running on sqftnuc1.  Need to add an extra check to environmentd.

**Comment by pchote** [comment]
Fixed by b65cecb7c3761dda1d290745a44660318166921b.
